.chloggen/disable-default-search-metrics.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'breaking'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: splunkenterprisereceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "disabled default metrics to prevent unexpected Splunk enterprise behavior"
```

Comment:
```suggestion
note: "disabled all metrics by default to ensure all searches run on Splunk enterprise are opt-in"
```

Reply:
this is a good callout -- was trying to be brief but maybe was a little _too_ brief